llvm/lib/Transforms/Instrumentation/RealtimeSanitizer.cpp

```diff
@@ -0,0 +1,50 @@
+#include "llvm/IR/Analysis.h"
+#include "llvm/IR/IRBuilder.h"
+#include "llvm/IR/Module.h"
+
+#include "llvm/Transforms/Instrumentation/RealtimeSanitizer.h"
+
+using namespace llvm;
+
+namespace {
```

Comment:
I believe llvm prefer `static` instead of `namespace {}`

Reply:
Fixed in latest